js-functions_basic-functions/js/index.js

```diff
@@ -37,5 +57,10 @@ Then log all variables to the console again.
 */
 
 // --v-- write your code here --v--
-
+function logBookData(bookTitle, author, rating, theNumberOfSales) {
+  console.log("title: ", bookTitle);
+  console.log("author: ", author);
+  console.log("rating: ", rating);
+  console.log("number of Sales: ", theNumberOfSales);
+}
```

Comment:
Just a tip how to do it in one line and with the special **\n** character:

console.log(`Title: ${title}\nAuthor: ${author}\nRating: ${rating}\nSales: ${numberSales}`);